docs/Using-Fleet/Fleet-desktop.md

```diff
@@ -32,7 +32,28 @@ For information on how to set the custom transparency link via a YAML configurat
 
 ## Securing Fleet Desktop
 
+Requests sent by Fleet Desktop and the web page that opens when clicking on the "My Device" tray item use a Random (Version 4) UUID token to uniquely identify each host.
```

Comment:
Is there a link to Random Version 4 that explains what it is??

Reply:
great idea, I linked to the RFC https://www.rfc-editor.org/rfc/rfc4122.html#section-4.4